IPython/terminal/debugger.py

```diff
@@ -86,5 +86,11 @@ def set_trace(frame=None):
 
 if __name__ == '__main__':
     import pdb
+    # IPython.core.debugger.Pdb.trace_dispatch shall not catch
+    # bdb.BdbQuit. When started through __main__ and an exeption
```

Comment:
thanks for pointing that out. Corrected in https://github.com/ipython/ipython/pull/9950/commits/3e812afa6e1f917695b517fc077ae54e5dab6c79